src/components/FormAlertWithSubmitButton.js

```diff
@@ -51,12 +55,14 @@ const FormAlertWithSubmitButton = props => (
         message={props.message}
         onFixTheErrorsPressed={props.onFixTheErrorsPressed}
     >
-        {isOffline => (isOffline ? (
+        {isOffline => ((isOffline && !props.enabledWhenOffline) ? (
```

Comment:
I don't get this... both sides of the ternary are rendering the exact same component now.

Reply:
🤦‍♂️, oops i didn't correct the logic after testing it out. I'm pushing the correct logic. which is basically reverting the button props and I tested it out.

Reply:
Updated and retested